src/Analyzers/Analyzers/src/Helpers/SymbolExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System.Diagnostics.CodeAnalysis;
```

Comment:
```suggestion
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

using System.Diagnostics.CodeAnalysis;
```

Reply:
This analyzer needs to be part of the component analyzers https://github.com/dotnet/aspnetcore/tree/main/src/Components/Analyzers